### PR TITLE
feat(devtools): use subset of Noto Sans for the logo and headings

### PR DESCRIPTION
**Pull request by Asuka109**
## Summary

image

Use subset of Noto Sans for the logo and headings instead of SVG images.

Generate the subset through https://font-subset.disidu.com/ to reduce font file size (2.6 MB -> 70 KB).

## Checklist

- [ ] I have added changeset via `pnpm run change`.
- [ ] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/devtools/client/src/styles/fonts.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/*
+ * This subset is modified from the Noto Sans font,
+ * which is distributed under the Open Font License.
+ * You can learn more about it at the following link.
+ * https://openfontlicense.org/
+ * https://fonts.google.com/noto/specimen/Noto+Sans
+ */
+@font-face {
+  font-family: 'NotoSans';
+  src: url('@/assets/NotoSans-latin-subset.ttf');
```
Comment (chenjiahan) [inline packages/devtools/client/src/styles/fonts.scss:10]: woff2 is smaller

Comment (chenjiahan) [reply packages/devtools/client/src/styles/fonts.scss:10]: can we use woff2

Comment (Asuka109) [reply packages/devtools/client/src/styles/fonts.scss]: Yep, I converted it to woff2 format, resulting in a significantly smaller size (70KB -> 38 KB).

**Review by chenjiahan — APPROVED** [review]

File: packages/devtools/client/src/styles/fonts.scss (hunk 1 above)
Comment (chenjiahan) [reply packages/devtools/client/src/styles/fonts.scss]: 👍

**Pull request merged**